Makefile

```diff
@@ -143,7 +150,9 @@ tarballs: $(ALL_PKGS_TARGETS)
 
 
 clean:
+	@if [ -t 0 ]; then read -p 'You are about to delete all untracked files! (use [Ctrl+C] to abort) '; fi
```

Comment:
This is a rather dangerous form of clean and will possibly cause someone to loose their work. I added a prompt if they are running this with a TTY so they can abort and this probably shouldn't block any existing uses of this target.

Reply:
I agree. I actually never use this make command